app.py

```diff
@@ -37,25 +53,33 @@ def __init__(self):
         self.notice_range_start = 0
         self.event_range_start = -1
 
-        with open("App_Data.txt", "r") as file:
-            try:
-                self.username = file.readline().strip()
-            except Exception:
-                self.username = ""
-            try:
-                self.password = file.readline().strip()
-            except Exception:
-                self.password = ""
-            try:
-                self.url = file.readline().strip()
-            except Exception:
-                self.url = ""
-            try:
-                self.delay_reload = int(file.readline().strip())
-            except Exception:
-                self.delay_reload = 5
-
-        self.window.after(self.delay_reload * 60000, self.reload)
+        try:
+            with open("App_Data.txt", "r") as file:
```

Comment:
It's a file that your data from the settings tab gets saved to. For example, it could get set to:
```
first.last@education.nsw.gov.au
SuperSecretPassword101
https://myschool.sentral.com.au/portal/dashboard
5.0
```

In order, this is:
- Username
- Password
- URL
- Reload time

If you want, I can change this to a .json file.

Reply:
I'll also comment on it

Reply:
Ah yeah it seemed something along the lines of that, I was just wondering because I'd never seen the file name before... is it just the same as `Sentral_Details.json`?

Reply:
Yeah, except that's for when you are using it for programming purposes instead of a UI. I.e. For programmers only as opposed to everyone.